src/components/Marketplace/ProduceCard.js

```diff
@@ -78,13 +88,17 @@ const useStyles = makeStyles({
 
 export default function ProduceCard(props) {
   const {
-    cropName, farmID, unitPrice, unitType,
+    // eslint-disable-next-line no-unused-vars
+    produceID, farmID, unitPrice, unitType, season, // (TODO: remove later when season is used)
   } = props;
   const [farmName, setFarmName] = useState('');
+  const [produceName, setProduceName] = useState('');
+  // eslint-disable-next-line no-unused-vars
+  const [produceCategory, setProduceCategory] = useState(''); // (TODO: remove later when used)
```

Comment:
What does remove later when used mean?

Reply:
I'm not currently using produceCategory for anything, since rn I'm just getting the information to use later for filtering. I added the //eslint line above so that eslint would allow me to compile even though it's not being used, but I put TODO: remove later to remember that I need to remove the eslint disable line once I use produceCategory.